src/store/Account.ts

```diff
@@ -406,8 +412,8 @@ export default {
                     .getMultisigAccountGraphInfo(currentAccountAddress)
                     .pipe(
                         map((g) => {
-                            // sorted array to be represented in multisig tree
                             commit('multisigAccountGraph', g.multisigEntries);
+                            commit('multisigAccountGraphInfo', MultisigService.getMultisigInfoFromMultisigGraphInfo(g));
                             return MultisigService.getMultisigInfoFromMultisigGraphInfo(g);
```

Comment:
it called the same function twice, 
`const x = MultisigService.getMultisigInfoFromMultisigGraphInfo(g)` will look clean.